sympy/assumptions/ask.py

```diff
@@ -1423,6 +1423,7 @@ def get_known_facts_dict():
 _val_template = 'sympy.assumptions.handlers.%s'
 _handlers = [
     ("antihermitian",     "sets.AskAntiHermitianHandler"),
+    ("antihermitian",     "matrices.AskAntiHermitianHandler"),
```

Comment:
Hi @debugger22 
If you look at [assume.py#L153](https://github.com/sympy/sympy/blob/master/sympy/assumptions/assume.py#L153), then handler for each predicate is a list.

In [ask.py#L1344](https://github.com/sympy/sympy/blob/master/sympy/assumptions/ask.py#L1344), we add handlers for each predicate.

Basically, it is a dictionary of "predicate" and "list of handlers". 
[`eval` function iterates over all the handlers of predicate and checks if it matches with the class of the argument](https://github.com/sympy/sympy/blob/master/sympy/assumptions/assume.py#L171)

This totally works, as If have added some tests too. Having said that, I'm not sure if this is the correct way to do this.
cc: @asmeurer @certik